cmd/lhsm-plugin-az/main.go

```diff
@@ -54,7 +56,9 @@ type (
 	azConfig struct {
 		NumThreads            int        `hcl:"num_threads"`
 		AzStorageAccount      string     `hcl:"az_storage_account"`
```

Comment:
Why is AzStorageAccount needed if we get the full URL?

Reply:
Not necessary, I retained it in case we need it for logging, or for other purpose. Can be discarded, but it is not a overhead, just wanted to keep it.